### PR TITLE
chore: use swc instead of ts-jest for some packages

### PR DESCRIPTION
**Pull request by Jolg42**
See https://github.com/pnpm/pnpm/issues/2174

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,9 +86,9 @@
     "util": "0.12.4"
   },
   "lint-staged": {
-    "*.{md,yml,yaml,json}": "prettier --write",
```
Comment (Jolg42) [inline package.json:88]: Only yaml file is pnpm-lock

File: packages/sdk/package.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
   "scripts": {
     "dev": "DEV=true node -r esbuild-register helpers/build.ts",
     "build": "node -r esbuild-register helpers/build.ts",
-    "prepublishOnly": "pnpm run build && pnpm run test",
+    "prepublishOnly": "pnpm run build",
```
Comment (millsp) [inline packages/sdk/package.json:24]: Could we remove this too as we already did a build before this?

Comment (Jolg42) [reply packages/sdk/package.json:24]: Out of scope of this PR I would say

Comment (millsp) [reply packages/sdk/package.json:24]: Fair enough :) I'll come with a followup PR

File: package.json [hunk 3]
```diff
@@ -73,6 +73,7 @@
     "prettier": "2.5.1",
     "redis": "3.1.2",
     "redis-lock": "0.1.4",
+    "regenerator-runtime": "^0.13.9",
```
Comment (millsp) [inline package.json:76]:
```suggestion
    "regenerator-runtime": "0.13.9",
```

**Review by millsp — APPROVED** [review]
Nice cleanup and test boost :rocket: Thank you!

File: package.json (hunk 3 above)
Comment (Jolg42) [reply package.json:76]: I'll let renovate do it, because it's going to need a lockfile update

**Pull request merged**